stock_move_location/wizard/stock_move_location.py

```diff
@@ -207,9 +209,13 @@ def _get_move_values(self, picking, lines):
     def _create_move(self, picking, lines):
         self.ensure_one()
         move = self.env["stock.move"].create(self._get_move_values(picking, lines))
-        if not self.env.context.get("planned"):
-            for line in lines:
-                line.create_move_lines(picking, move)
+        for line in lines:
+            line.create_move_lines(picking, move)
+        if self.env.context.get("planned"):
```

Comment:
To have the logic consolidated in one place for 'planned' transfers.

```suggestion
        if self.env.context.get("planned"):
            for line in lines:
                available_quantity = self.env["stock.quant"]._get_available_quantity(
                    line.product_id,
                    line.origin_location_id,
                    lot_id=line.lot_id,
                    strict=False,
                )
                move._update_reserved_quantity(
                    line.move_quantity,
                    available_quantity,
                    line.origin_location_id,
                    lot_id=line.lot_id,
                    strict=False,
                )
```